messages/en.json

```diff
@@ -61,6 +60,99 @@
+    "example_text": "Lorem ipsum dolor sit amet. Aut rerum illum ut voluptatem enim qui consequatur exercitationem nam molestias fugit est expedita voluptas. Ad quas reiciendis non voluptas recusandae qui optio ratione."
+  },
+  "leaderboard": {
+    "leaderboard": "Leaderboard",
+    "aboutLeaderboard": "The leadership consists of a Leader, Deputy Leader, and Financial Officer in Hackerspace. They ensure that Hackerspace NTNU operates optimally and make decisions and planning for operations with the board.",
+    "leader": "Leader",
+    "deputyLeader": "Deputy Leader",
+    "financialManager": "Financial Manager"
+  },
+  "member_representative": {
+    "information": "<p1> Hackerspace shall have a representative who is appointed during the general assembly. This representative has no voting rights on the board but is a contact person for Hackerspace members regarding issues they do not wish to address directly with the board. The representative has a duty of confidentiality and will, if necessary, relay issues anonymized to the board. </p1> <p2> If you wish to get in touch, the representative is available on Slack. </p2>",
+    "question1": "",
+    "answer1": "",
+    "question2": "",
+    "answer2": "",
+    "question3": "",
+    "answer3": ""
+  },
+  "devops": {
+    "information": "The main tasks of DevOps are to develop and manage Hackerspace's website, but we also work on the operation of infrastructure, databases, servers, and are responsible for various gadgets.",
+    "question1": "Does DevOps requires any programming-knowledge before becoming a member? ",
+    "answer1": "You don't need any programming-knowledge before joing our group. Our group are welcome for everybody who wants to learn. As a new member, you'll go through our 'get-to-know'-program that will show you our prosedure and technoliges that we use. But don't worry, you can take everything in your own tempo!",
+    "question2": "What is the main focus for DevOps this semester?",
+    "answer2": "We are focusing on launching a new website for Hackerspace. The technologies that we use is ....",
+    "question3": "Can I choose what I want to work with?",
+    "answer3": "Yes, of course everybody get to choose the task they want to work with. Prior to our focus on luanching new website, the tasks wil be related to this. But you can also pitche an idea that you want to implement on our website and hear what the rest of the groups think about it.",
+    "leader": "Leader",
+    "deputyLeader": "Deputy Leader",
+    "member": "Member"
+  },
+  "labops": {
+    "aboutLabOps": "The main tasks of LabOps are to run Hackerspace's premises, plan and carry out events, hold stands and run other promotions.",
+    "question1": "",
+    "answer1": "",
+    "question2": "",
+    "answer2": "",
+    "question3": "",
+    "answer3": "",
+    "leader": "Leader",
+    "deputyLeader": "Deputy Leader",
+    "member": "Member"
+  },
+  "breadboard_computer": {
```

Comment:
I feel like all "temporary" groups should have the Group prefix while permanent groups like labops and devops should not